benchmarks/span/scenario.py

```diff
@@ -24,7 +30,7 @@ def run(self):
         def _(loops):
             for _ in range(loops):
                 for i in range(self.nspans):
-                    s = utils.gen_span("test." + str(i))
```

Comment:
We already have a benchmark scenario to cover `tracer.trace()` here: https://github.com/DataDog/dd-trace-py/blob/d4343d340b48e37f5e5f6afa5656ae65043404c5/benchmarks/tracer/scenario.py#L25

Reply:
ahhh I'll close this PR